src/transformers/models/switch_transformers/modeling_switch_transformers.py

```diff
@@ -127,6 +127,24 @@ def load_balancing_loss_func(router_probs: torch.Tensor, expert_indices: torch.T
     return torch.mean(tokens_per_group_and_expert * router_prob_per_group_and_expert) * (num_experts**2)
 
 
+class ClassificationHead(nn.Module):
+    """Head for sentence-level classification tasks."""
+
+    def __init__(self, config: SwitchTransformersConfig, model_dim: int):
+        super().__init__()
+        self.dense = nn.Linear(model_dim, model_dim, bias=False)
+        self.dropout = nn.Dropout(p=0.1)
+        self.out_proj = nn.Linear(model_dim, config.num_labels, bias=False)
+
+    def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
+        hidden_states = self.dropout(hidden_states)
+        hidden_states = self.dense(hidden_states)
+        hidden_states = torch.tanh(hidden_states)
+        hidden_states = self.dropout(hidden_states)
+        hidden_states = self.out_proj(hidden_states)
+        return hidden_states
```

Comment:
```suggestion
# Copied from transformers.models.t5.modeling_t5.T5ClassificationHead with T5->SwitchTransformers
class SwtichTransformersClassificationHead(nn.Module):
    """Head for sentence-level classification tasks."""

    def __init__(self, config: SwitchTransformersConfig, model_dim: int):
        super().__init__()
        self.dense = nn.Linear(model_dim, model_dim, bias=False)
        self.dropout = nn.Dropout(p=0.1)
        self.out_proj = nn.Linear(model_dim, config.num_labels, bias=False)

    def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
        hidden_states = self.dropout(hidden_states)
        hidden_states = self.dense(hidden_states)
        hidden_states = torch.tanh(hidden_states)
        hidden_states = self.dropout(hidden_states)
        hidden_states = self.out_proj(hidden_states)
        return hidden_states
```

Reply:
Let's add a copied from statement here as the module is a copy of `T5ClassificationHead`